spec/system/administrateurs/procedure_attestation_template_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'system/administrateurs/procedure_spec_helper'
+
+describe 'As an administrateur I wanna manage the procedure\'s attestation_tempalte', js: true do
+  include ProcedureSpecHelper
+
+  let(:administrateur) { create(:administrateur) }
+
+  let!(:procedure) do
```

Comment:
J'ai l'impression que `let!` est généralement un code-smell (les créations implicites s'enchainent dans un ordre pas forcément prédictible, et parfois créent des records inutilement). Ici la procedure va en plus être créée au moment de l'appel à login_as, si je comprends bien.

```suggestion
  let(:procedure) do
```